Marlin/src/lcd/ultralcd.cpp

```diff
@@ -71,6 +71,8 @@
 #if ENABLED(AUTO_BED_LEVELING_UBL) || ENABLED(G26_MESH_VALIDATION)
   bool lcd_external_control; // = false
 #endif
+  
+  int IS_SOFT_ENDSTOP_ENABLED = 1;
 
```

Comment:
That's an upper case variable, not a define. Should be lower case.

Reply:
Yeah, you are right, it should be in lower case though. Thanks for pointing this out!

Reply:
No need to add this. The global `soft_endstops_enabled` is already defined in `motion.h`.